diff -up magicolor2430DL-1.6.1/src/rastertokmlf.c.omv~ magicolor2430DL-1.6.1/src/rastertokmlf.c
--- magicolor2430DL-1.6.1/src/rastertokmlf.c.omv~	2018-11-27 18:35:11.186713516 +0100
+++ magicolor2430DL-1.6.1/src/rastertokmlf.c	2018-11-27 18:35:21.066769903 +0100
@@ -53,6 +53,7 @@
  */
 
 #include <cups/cups.h>
+#include <cups/ppd.h>
 //#include <cups/string.h>
 #include <cups/language.h>
 #include <cups/raster.h>
